library/src/test/java/com/mapzen/valhalla/RouterTest.java

```diff
@@ -45,56 +46,59 @@ public void tearDown() throws Exception {
     }
 
     @Test
-    public void shouldDefaultToEnUs() throws Exception {
-        assertThat(router.getJSONRequest().directionsOptions.language).contains("en-US");
+    public void shouldUseDefaultLocaleIfNoLanguageSpecified() throws Exception {
+        Locale.setDefault(Locale.FRANCE);
+        double[] loc = new double[] {1.0, 2.0};
+        router = new ValhallaRouter().setLocation(loc).setLocation(loc);
```

Comment:
The router throws an exception if you attempt to fetch a route with fewer than two locations.